typescript/infra/package.json

```diff
@@ -22,6 +22,7 @@
     "@solana/web3.js": "^1.78.0",
     "asn1.js": "5.4.1",
     "aws-kms-ethers-signer": "^0.1.3",
+    "deep-object-diff": "^1.1.9",
```

Comment:
This package has zero deps https://www.npmjs.com/package/deep-object-diff, small bundle size https://bundlephobia.com/package/deep-object-diff@1.1.9